algorand-sdk.cabal

```diff
@@ -46,14 +47,48 @@ library
       Paths_algorand_sdk
   hs-source-dirs:
       src
-  default-extensions: AllowAmbiguousTypes BangPatterns ConstraintKinds DataKinds DefaultSignatures DeriveDataTypeable DeriveGeneric DerivingStrategies DuplicateRecordFields EmptyCase FlexibleContexts FlexibleInstances FunctionalDependencies GADTs GeneralizedNewtypeDeriving LambdaCase MultiParamTypeClasses MultiWayIf NamedFieldPuns OverloadedStrings OverloadedLabels PatternSynonyms RankNTypes RecordWildCards ScopedTypeVariables StandaloneDeriving TemplateHaskell TupleSections TypeFamilies TypeOperators UndecidableInstances ViewPatterns TypeApplications
+  default-extensions:
+      AllowAmbiguousTypes
+      BangPatterns
+      ConstraintKinds
+      DataKinds
+      DefaultSignatures
+      DeriveDataTypeable
+      DeriveGeneric
+      DerivingStrategies
+      DuplicateRecordFields
+      EmptyCase
+      FlexibleContexts
+      FlexibleInstances
+      FunctionalDependencies
+      GADTs
+      GeneralizedNewtypeDeriving
+      LambdaCase
+      MultiParamTypeClasses
+      MultiWayIf
+      NamedFieldPuns
+      OverloadedStrings
+      OverloadedLabels
+      PatternSynonyms
+      RankNTypes
+      RecordWildCards
+      ScopedTypeVariables
+      StandaloneDeriving
+      TemplateHaskell
+      TupleSections
+      TypeFamilies
+      TypeOperators
+      UndecidableInstances
+      ViewPatterns
+      TypeApplications
   ghc-options: -Wall -Wcompat -Wincomplete-record-updates -Wincomplete-uni-patterns -Wredundant-constraints
   build-depends:
-      aeson >=1.3 && <1.5
+      aeson >=1.3 && <1.6
     , aeson-casing >=0.1.1.0 && <0.3
     , base >=4.7 && <4.15
-    , base32 >=0.2 && <0.3
+    , base32 ==0.2.*
```

Comment:
Erm, it looks strange, why did this constraint become stricter? AFAIU there is `base32 >= 0.2 && < 0.3` in `package.yaml`, why is it different in the generated cabal file? Could you please investigate?

Reply:
I think it's because `0.2.*` is all versions before `0.3`, so this statements equivalent

Reply:
:man_facepalming: 

You are right, it was stupid, they are indeed equivalent. So probably it's just a chance in newer version of `hpack`.